ee/tabby-webserver/src/repositories/mod.rs

```diff
@@ -40,12 +40,11 @@ async fn not_found() -> StatusCode {
     StatusCode::NOT_FOUND
 }
 
-#[instrument(skip(repo))]
```

Comment:
add back this. as `#[instrument(skip(rs))]`

and you can remove the `Debug` implementation for RepositoryCache.